Hangman/Hangman/main.swift

```diff
@@ -7,6 +7,184 @@
+        """)
+    } else if guessMax == 3 {
+        print("""
+            _______
+            |/      |
+            |      (_)
+            |       |
+            |
+            |
+            |
+           _|___
+        """)
+    } else if guessMax == 2 {
+        print("""
+            _______
+            |/      |
+            |      (_)
+            |       |
+            |       |
+            |
+            |
+           _|___
+        """)
+    } else if guessMax == 1 {
+        print("""
+            _______
+            |/      |
+            |      (_)
+            |      \\|/
+            |       |
+            |
+            |
+           _|___
+        """)
+    } else if guessMax == 0 {
+       print("""
+            _______
+            |/      |
+            |      (_)
+            |      \\|/
+            |       |
+            |      / \\
+            |
+           _|___
+        """)
+        sleep(3)
+        print("Game Over")
+    }
+}
+    
+
+
+func hangMan() {
+    let randomWord = wordsBank.randomElement() ?? "animal"
+    var hiddenWord: [Character] = Array(repeating: "_", count: randomWord.count)
+    
+    var indices = [Int]()
+    
+func word() {
+        for char in hiddenWord {
+            print(char, terminator: "")
+        }
+        print()
+    }
+    
+    repeat {
+        //print(String(hiddenWord.joined()))
+        hangManArt()
+        print("You have \(guessMax) attempts left")
+        word()
+        //print(randomWord) // shows the actual word
+        print("Guess a latter a - z")
+        let enteredCharacter = readLine()?.lowercased() ?? "ha"
+        //let enteredCharecter1 = Character(enteredCharacter)
+        if enteredCharacter.count > 1 || !alphabet.contains(enteredCharacter) {
+            print("Please enter only Valid character!")
+            print()
+            continue
+        }
+//        } else if alphabetArray {
+//            print("This is not a part of the alphabet! Try again!")
+//    }
+    
+        for (index, char) in randomWord.enumerated() {
+            if enteredCharacter == String(char) {
+                indices.append(index)
+            }
+            
+        }
+        for (index, _) in hiddenWord.enumerated() {
+            if indices.contains(index) {
+                hiddenWord[index] = Character(enteredCharacter)
+            }
+        }
+        
+        if !randomWord.contains(enteredCharacter) {
+            guessMax -= 1
+        }
+        if !hiddenWord.contains("_") {
+            print("You win!")
+            sleep(2)
+            print("do you want to try again?")
+            print("Yes/No")
+            let continueGame = readLine()?.lowercased() ?? "no"
+            
+            if continueGame == "yes" {
+                guessMax = 6
+                hangMan()
+            }
+            break
+        }
+        indices.removeAll()
+        
+        if guessMax == 0 {
+            print("You loose 😭")
+            sleep(2)
```

Comment:
You could have included a second line of code here that displayed what the hidden/random word was. Remember that one of the requirements was to display the hidden/random word once the user lost.